### PR TITLE
Insights docs

### PR DESCRIPTION
**Pull request by matheuslc**
# Description
This PR adds a new page within the Insights docs. We moved the docs previously named `Query stats` to a new page called Insights.  

We are releasing new features related to the query stats page, and this PR keeps the docs up to date with the latest features.

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [x] Is the content technically accurate?
*   [x] Is the content complete?
*   [x] Is the content presented in a logical order?
*   [x] Does the content use appropriate names for features and products?
*   [x] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [x] Is the content free from typos?
*   [x] Does the content use plain English?
*   [x] Does the content contain clear sections for concepts, tasks, and references?
*   [x] Have any images been uploaded to the correct location, and are resolvable?
*   [x] If the page index was updated, are redirects required
      and have they been implemented?
*   [x] Have you checked the built version of this content?

### CODE REVIEW

File: use-timescale/metrics-logging/insights.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+cloud_ui:
+    path:
+        - [services, :serviceId, insights]
+---
+
+# Insights
+
+Insights allows you to gain a comprehensive understanding of how your queries perform over time. It empowers you to optimize your queries and make the most efficient use of your instance resources.
+
+You can view query-level insights from the `Insights` tab at your service Cloud Dashboard.
```
Comment (vineethapai) [inline use-timescale/metrics-logging/insights.md:16]: Should we say this tool provides insights on queries performed in the last 24 hours?

**Review by vineethapai — CHANGES_REQUESTED** [review]

File: use-timescale/metrics-logging/insights.md (hunk 1 above)
Comment (matheuslc) [reply use-timescale/metrics-logging/insights.md:16]: I think we should, good idea! At least clarify the maximum date range we currently support!

Comment (matheuslc) [reply use-timescale/metrics-logging/insights.md:16]: @vineethapai, what do you think to rephrase to: You can view up to 24 hours of query-level insights from the `Insights` tab on your service Cloud Dashboard.

Comment (vineethapai) [reply use-timescale/metrics-logging/insights.md]: Can we say this "On the insights tab, you can access insights on queries and instance resources for the previous 24 hours."

Comment (matheuslc) [reply use-timescale/metrics-logging/insights.md]: For sure! I'll change it right now

Comment (matheuslc) [reply use-timescale/metrics-logging/insights.md]: Done ✅ Waiting the preview just to confirm, but the change is already done

**Review by vineethapai — APPROVED** [review]

**Pull request merged**